src/yoroi-wallets/cardano/types.ts

```diff
@@ -336,4 +343,11 @@ const yoroiWalletKeys: Array<YoroiWalletKeys> = [
   'toJSON',
   'fetchCurrentPrice',
   'fetchFundInfo',
+  'internalAddresses',
+  'externalAddresses',
+  'confirmationCounts',
```

Comment:
`confirmationCounts` is not in use anymore (calculated on tx details)

Reply:
its used to derive `TransactionInfo`